vllm/spec_decode/ngram_worker.py

```diff
@@ -133,12 +136,15 @@ def sampler_output(
     def get_spec_proposals(
         self,
         execute_model_req: ExecuteModelRequest,
+        # Unused parameter. NGramWorker does not use the KV Cache and
+        # therefore does not need this parameter.
+        seq_ids_with_bonus_token_in_last_step: set = set(),
```

Comment:
btw this is bad python antipattern; the object will be re-used over the lifetime of the function, see https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments

Reply:
Done